packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -220,7 +221,15 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     }
   }
 
-  async function handlePublish(publishDate: Date, updateDate: Date) {
+  async function handlePublish(publishDate: Date, updateDate: Date, noSlug: boolean) {
+    if (noSlug) {
+      Notification.error({
```

Comment:
Use Alert instead of Notification and don't set a duration that makes it stay until the user closes it.